clients/drcachesim/common/trace_entry.h

```diff
@@ -476,8 +476,11 @@ typedef enum {
     TRACE_MARKER_TYPE_CHUNK_INSTR_COUNT,
 
     /**
-     * Marks the end of a chunk.  The final chunk does not have such a marker
-     * but instead relies on the #TRACE_TYPE_FOOTER entry.
+     * Marks the end of a chunk.  The value of this marker is the chunk ordinal, which
+     * is an increasing counter set by #dynamorio::drmemtrace::record_filter_t starting
```

Comment:
No, for 99% of traces record_filter is *not* involved at all (raw2trace inserts this during normal post-processing).  Please remove this part.

Reply:
Removed.